### PR TITLE
chore: bump version of near-workspaces

### PR DESCRIPTION
**Pull request by iho**

### CODE REVIEW

File: examples/callback-results/Cargo.toml [hunk 1]
```diff
@@ -8,13 +8,17 @@ edition = "2021"
...
-near-sdk = { path = "../../near-sdk" }
+near-sdk = "4.1.1"
 
 [dev-dependencies]
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
 anyhow = "1.0"
+near-gas = { version = "*" }
```
Comment (frol) [inline examples/callback-results/Cargo.toml:18]: `Gas` is exposed through [`near_sdk::types::Gas`](https://github.com/near/near-sdk-rs/blob/e4c924058e9159ff54a5bfc4adaa8b79ba04d876/near-sdk/src/types/mod.rs#L13); do not import it separately.

File: examples/factory-contract/Cargo.toml [hunk 2]
```diff
@@ -6,16 +6,20 @@ edition = "2021"
 
 [dev-dependencies]
 anyhow = "1.0"
-near-sdk = { path = "../../near-sdk" }
+near-sdk = { version = "4.1.1", features = ["unstable"] }
```
Comment (frol) [inline examples/factory-contract/Cargo.toml:9]: What `unstable` feature is used in this example? Examples should avoid using unstable features unless they specifically want to feature that specific feature.

File: examples/fungible-token/Cargo.toml [hunk 3]
```diff
@@ -8,16 +8,21 @@ edition = "2021"
 anyhow = "1.0"
 base64 = "0.13"
 near-primitives = "0.17"
-near-sdk = { path = "../../near-sdk" }
+# near-sdk = { path = "../../near-sdk" }
```
Comment (frol) [inline examples/fungible-token/Cargo.toml:11]: clean it up, please

File: examples/fungible-token/Cargo.toml [hunk 4]
```diff
@@ -8,16 +8,21 @@ edition = "2021"
...
+near-sdk = { version = "4.1.1" }
 near-units = "0.2.0"
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
+near-gas = { version = "*" }
```
Comment (frol) [inline examples/fungible-token/Cargo.toml:17]: ^

File: examples/fungible-token/Cargo.toml [hunk 5]
```diff
@@ -29,7 +34,4 @@ overflow-checks = true
 
 [workspace]
 # remember to include a member for each contract
-members = [
-  "ft",
-  "test-contract-defi",
-]
+members = ["ft", "test-contract-defi"]
```
Comment (frol) [inline examples/fungible-token/Cargo.toml:37]: Could you, please, avoid formatting parts that you don't touch?

File: examples/lockable-fungible-token/Cargo.toml [hunk 6]
```diff
@@ -8,16 +8,20 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = { path = "../../near-sdk" }
+near-sdk = { version = "4.1.1", features = ["unstable"] }
```
Comment (frol) [inline examples/lockable-fungible-token/Cargo.toml:11]: unnecessary "unstable"

File: examples/lockable-fungible-token/Cargo.toml [hunk 7]
```diff
@@ -8,16 +8,20 @@ edition = "2021"
...
+
 
 [dev-dependencies]
 anyhow = "1.0"
 near-primitives = "0.17"
-near-sdk = { path = "../../near-sdk" }
+near-sdk = { version = "4.1.1", features = ["unstable"] }
```
Comment (frol) [inline examples/lockable-fungible-token/Cargo.toml:17]: ^

File: examples/non-fungible-token/Cargo.toml [hunk 8]
```diff
@@ -12,7 +12,8 @@ near-sdk = { path = "../../near-sdk" }
 near-units = "0.2.0"
 serde_json = "1.0"
 tokio = { version = "1.14", features = ["full"] }
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
+near-gas = "0.2.3"
```
Comment (frol) [inline examples/non-fungible-token/Cargo.toml:16]: ^

File: near-contract-standards/Cargo.toml [hunk 9]
```diff
@@ -13,7 +13,9 @@ NEAR smart contracts standard library.
 """
 
 [dependencies]
-near-sdk = { path = "../near-sdk", version = "~4.1.1", default-features = false, features = ["legacy"] }
+near-sdk = { path = "../near-sdk", default-features = false, features = [
+    "legacy",
+] }
```
Comment (frol) [inline near-contract-standards/Cargo.toml:18]: Let's revert it

File: near-sdk/Cargo.toml [hunk 10]
```diff
@@ -21,7 +21,7 @@ required-features = ["abi", "unstable"]
 # Provide near_bidgen macros.
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-near-sdk-macros = { path = "../near-sdk-macros", version = "~4.1.1" }
+near-sdk-macros = { path = "../near-sdk-macros" }
```
Comment (frol) [inline near-sdk/Cargo.toml:24]: Let's revert it.

File: examples/adder/Cargo.lock
Comment (frol) [inline examples/adder/Cargo.lock:1]: Let's remove and gitignore Cargo.lock files in examples while we are at it: https://github.com/near/near-sdk-rs/pull/1093

**Review by frol — CHANGES_REQUESTED** [review]

File: examples/cross-contract-calls/Cargo.toml [hunk 11]
```diff
@@ -11,11 +11,15 @@ near-units = "0.2.0"
...
 tokio = { version = "1.14", features = ["full"] }
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
 
 cross-contract-high-level = { path = "./high-level" }
 cross-contract-low-level = { path = "./low-level" }
 
+
+[patch.crates-io]
+near-sdk = { path = "../../near-sdk" }
```
Comment (dj8yfo) [inline examples/cross-contract-calls/Cargo.toml:21]: Line 9 probably also has to be changed to `version` to follow the pattern

File: near-sdk/src/store/tree_map/entry.rs [hunk 12]
```diff
@@ -135,6 +135,7 @@ where
     /// assert_eq!(map["poneyland"], None);
     /// # }
     /// ```
+    #[allow(clippy::unwrap_or_default)]
```
Comment (frol) [inline near-sdk/src/store/tree_map/entry.rs:138]: Let's move it to the actual line of code instead of applying to the whole function

**Review by frol — CHANGES_REQUESTED** [review]

File: examples/cross-contract-calls/Cargo.toml (hunk 11 above)
Comment (iho) [reply examples/cross-contract-calls/Cargo.toml:21]: done

File: .github/workflows/test_examples.yml [hunk 13]
```diff
@@ -14,12 +14,11 @@ jobs:
       matrix:
         platform: [ubuntu-latest, macos-latest]
         # after fix of https://github.com/near/nearcore/pull/9140#issuecomment-1576655073, it should be [stable]
-        toolchain: [1.69.0]
+        toolchain: [1.70.0]
```
Comment (dj8yfo) [inline .github/workflows/test_examples.yml:17]: @frol should it be stable according to comment on 16?

File: examples/non-fungible-token/Cargo.toml [hunk 14]
```diff
@@ -23,6 +23,9 @@ debug = false
 panic = "abort"
 overflow-checks = true
 
+[patch.crates-io]
+near-sdk = { path = "../../near-sdk" }
+
```
Comment (dj8yfo) [inline examples/non-fungible-token/Cargo.toml:28]: similarly `version` probably skipped here on line 11

Comment (iho) [reply examples/non-fungible-token/Cargo.toml:28]: Thanks. Done

File: .github/workflows/test_examples.yml (hunk 13 above)
Comment (iho) [reply .github/workflows/test_examples.yml]: Change to stable. Thanks

**Review by frol — APPROVED** [review]

**Pull request merged**